src/web/Dockerfile

```diff
@@ -57,10 +63,23 @@ ENV NEXT_PUBLIC_SUPABASE_URL ${SUPABASE_URL}
 ARG ANON_KEY
 ENV NEXT_PUBLIC_ANON_KEY ${ANON_KEY}
 
-RUN npm install --no-package-lock
+RUN pnpm install
+
+RUN pnpm build
+
+## Telescope docs
+WORKDIR /docusaurus
+
+RUN pnpm install
+
+RUN pnpm build
+
+## Deploy ######################################################################
+FROM nginx:1.20.2-alpine as deploy
+
+WORKDIR /
 
-RUN npm run build
+COPY --from=dependencies /app/out /var/www/data
 
-FROM nginx:stable-alpine as build
+COPY --from=dependencies /docusaurus/build /usr/share/nginx/html
```

Comment:
```suggestion
COPY --from=dependencies /docusaurus/build /usr/share/nginx/html/docs
```
I got it to work with this change.

Reply:
It also works with:

### nginx file:
```nginx
  location /docs {
      root /var/www/data;
    }
```

### Dockerfile
```yml
COPY --from=dependencies /docusaurus/build /var/www/data/docs
```

Reply:
With this fix, I could see that the `baseUrl` also had to be changed to `/docs/` (`docusaurus` will warn you about this). Once you fix that:
![image](https://user-images.githubusercontent.com/23108901/161400112-4dcc02af-abfa-4c2c-b082-1fd40d244d45.png)

Reply:
Thank you so much! This worked